README.md

```diff
@@ -154,23 +155,14 @@ to.
 Property/parameter-level attribute for configuration value mapping.
 
 **Parameters:**
-- `path` (string, optional): Custom configuration path using dot notation
-(e.g., 'api.endpoint')
-- `required` (bool, optional): Whether the configuration value is required
-(default: false)
-
-## Configuration Structure
+- `path` (string, optional): Custom configuration path using dot notation (e.g., 'api.endpoint')
+- `required` (bool, optional): Whether the configuration value is required (default: false)
 
-Extension configuration in TYPO3 is typically stored in
-`config/system/settings.php` under the
-`$GLOBALS['TYPO3_CONF_VARS']['EXTENSIONS']` array, or in
-`config/system/additional.php`, or custom configurations.
+**Note**: Default values are defined as PHP constructor parameter defaults, not in the attribute.
 
-TYPO3's backend configuration interface allows administrators to modify these values, but all
-values set through the backend module will be stored as strings regardless of their intended type.
+## How It Works
 
-This package retrieves the configuration with `TYPO3\CMS\Core\Configuration\ExtensionConfiguration`
-regardless on how it got there.
+TYPO3 stores extension configuration in `$GLOBALS['TYPO3_CONF_VARS']['EXTENSIONS']` as strings. This extension automatically converts those strings to proper PHP types using your configuration class schema.
```

Comment:
```suggestion
TYPO3 stores extension configuration in `$GLOBALS['TYPO3_CONF_VARS']['EXTENSIONS']` as associative array with strings values.
This extension automatically converts those strings to proper PHP types using your configuration class schema.
```

Reply:
Ya, your fix is more precise. Incorporated in https://github.com/mteu/typo3-typed-extconf/pull/44/commits/c73dea09947c895a074aa7ee9bcb8f41209237fd. Thanks!